### ISSUE TITLE
appending hypervisor keys with `skywire-cli visor gen-config`

### ISSUE THREAD

**Issue by 0pcom** [skycoin/skywire#610]
**Feature description**

The most efficient way to interact with the config .json files is through the action of `skywire-cli visor gen-config` flags and arguments. With that in mind, there are a few predefined actions we can anticipate the user might want to do in terms of hypervisor configuration.

One thing I would like to be able to do is to __either__ append or overwrite the hypervisor key field.

Also, I would ideally like to be able to set a visor as it's own hypervisor as well as specifying a remote visor.

for my immediate purposes, being able to **append a key or keys** and **being able to reset the field** or better documentation as to when to expect this behavior

**Is your feature request related to a problem? Please describe.**

Making configuration and reconfiguraion easier

**Describe the solution you'd like**

for example
```
skywire-cli visor gen-config -r --hypervisor-keys-append 0x0000..
```
the above should add the key to the list of hypervisor keys
the name of the flag is a suggestion and not critical to me

```
skywire-cli visor gen-config -r --hypervisor-keys ""
```
It would be nice if resetting the hypervisor keys field was done like that

```
skywire-cli visor gen-config -r -o /opt/skywire/skywire-config.json
```
So when you do a vanilla regeneration like this it shouldn't reset your hypervisor keys. The `vanilla` mode of configuration generation should update the forma of the config file and if necessary change the default values to reflect whatever changes were made in the source.

The worst thing that can happen is that you just have to regenerate the configuration if something gets messed up, which is not so terrible. So it's not critical that what is there will always be retained between versions, etc. but it should be easy to put it back more or less how it was.

**Describe alternatives you've considered**

It seems like it wouldn't be too difficult to implement this and it seems like what is stated above is more or less the correct way to go about it. The only immediate alternative is to manually edit the config file.

**Comment by i-hate-nicknames** [comment]
It seems to me that it may make more sense to introduce another command, like `update-config`. Because at this point we do not generate a config anymore but work with an existing one. It should have one required argument that is the config file that we want to update. For now it will only have `append-hypervisor-keys` but in future maybe there will be others.

Otherwise we will have to either use multiple file arguments (input from which to read and output where to write), or use output for input purposes, as well as handle weird scenarios when input is not specified/absent but `append-hypervisor-keys` argument is present. What do you think?

**Comment by jdknives** [comment]
@0pcom @i-hate-nicknames updating the config file via another command seems better than reusing the `gen-config` command as it is more intuitive and can be more selective. 

In addition to appending or removing hypervisor keys, something like swapping out only test and prod deployments URLs would be good to have to be able to switch between the two deployments without having to regenerate the config.

**Comment by jdknives** [comment]
Fixed in #621
